lib/osf-components/addon/components/storage-provider-manager/osf-storage-manager/component.ts

```diff
@@ -60,19 +61,25 @@ export default class OsfStorageManager extends Component<Args> {
     @task
     @waitFor
     async goToFolder(folder: OsfStorageFile) {
+        const parentFolder = await folder.fileModel.parentFolder;
```

Comment:
I've been giving this some thought, and I'm thinking maybe we should make a breadcrumbs property for the manager, which is a list of folders. The first item would be the root folder, and the last item would be the current folder. This way, we 1) generalize folder navigation across all providers; and 2) have a way of generating visible breadcrumbs above the file list. 

Thoughts?

Reply:
(I'm okay with this being done in a separate story, but I'm still interested in your thoughts on the subject.)

Reply:
I like the idea.